src/main/resources/blueprints/minecolonies/medievalbirch/pack.json

```diff
@@ -4,7 +4,7 @@
   "authors": [
     "Carlansor"
   ],
-  "desc": "[Intermediate] Hear ye! A new castle emerges! Find a Good spot near a river and a forest and build your own medieval town! Alt townhall is a castle and regular one’s a manor, some buildings are better placed near water and mind the size of the university and library! Those usually go well placed together.",
```

Comment:
Note that `'s` is a shorthand for "is", so the original text was techically correct